example/src/main/java/com/stripe/example/activity/PaymentSessionActivity.java

```diff
@@ -181,6 +254,61 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     protected void onDestroy() {
         super.onDestroy();
         mPaymentSession.onDestroy();
+        LocalBroadcastManager.getInstance(this).unregisterReceiver(mBroadcastReceiver);
+
+    }
+
+    /**
+     * Converts an integer price in the lowest currency denomination to a Google string value.
+     * For instance (100, USD) -> "1.00", but (100, JPY) -> "100"
+     * @param price the price in the lowest available currency denomination
+     * @param currency the {@link Currency} used to determine how many digits after the decimal
+     * @return a String that can be used as an Android Pay price string
+     */
+    @NonNull
+    public static String getPriceString(Long price, @NonNull Currency currency) {
```

Comment:
any reason why the existing the PaymentUtils.formatPriceString doesn't work?

preference for formatPriceString because it handles rendering the currency of the amount in the question in the convention of the locale of the users (also, it mostly uses pre-existing android formatters so we don't have to maintain it)

Though, do we need this one if we're doing to add AndroidPay into the example?

Reply:
Thought this was in a different PR -- I switched it to use the value in the code that I moved into stripe-android. The only reason not to use PaymentUtils.formatPriceString is that the methods aren't public. I don't think the example project is a good enough reason to make them public, so I'll use the google-y version.